README.md

```diff
@@ -24,7 +24,8 @@ The GitHub Rocket.Chat App provides a seamless integration between GitHub and Ro
 <h2 align='center'>💡 Usage 💡</h2>
 <ul>
     <li> See Helper Message / Command List -> /github help </li>
-    <li> The app can be accessed with any of the slash commands /gh or /github </li>
+    <li> The app can be accessed with any of the slash commands /gh or /github </li>  
```

Comment:
@VipinDevelops These two should not be in bullet points.  Can you please fix these ?

Reply:
I tied to impove the overall style of command list here is new one do let me know if any changes required 
![image](https://github.com/RocketChat/Apps.Github22/assets/99081689/0d8c966f-12d6-43e7-9bdb-1eb0e7c4d7f5)